demos/question-answering/README.md

```diff
@@ -1,17 +1,32 @@
 # Question Answering
 
-This project utilizes an open-source Large Language Model (LLM) that is capable of answering questions over a corpus of private documentation. To achieve this, the experiment employs a Vector Store that captures and indexes a latent representation for each document. This allows the application to retrieve the relevant context based on the user's questions, enabling accurate and efficient question answering.
+This project utilizes an open-source Large Language Model (LLM) that is capable of answering questions over a corpus of
+private documentation. To achieve this, the experiment employs a Vector Store that captures and indexes a latent
```

Comment:
This project utilizes an open-source Large Language Model (LLM) that is capable of answering questions over a corpus of private documentations. The documents are indexed and stored as latent representations in a Vector Store. This allows the application to retrieve the relevant documents based on the user's
questions, enabling accurate and efficient question answering.